tools/tgentools/analysis.py

```diff
@@ -100,7 +101,7 @@ def save(self, filename=None, output_prefix=os.getcwd(), do_compress=True):
 
         logging.info("saving analysis results to {}".format(filepath))
 
-        outf = util.FileWritable(filepath, do_compress=do_compress)
+        outf = util.FileWritable(filepath, do_compress=do_compress, xz_nthreads=self.num_subprocs)
```

Comment:
If you have 10 files to analyze and 64 cpu cores, does this mean it will create 640 simultaneous xz threads? This is probably fine, but just thought I'd mention it.

Reply:
I think how it works is that the `analyze` func is called first using 64 sub processes in order to parse all of the tgen files in parallel, and then when the analysis is done all of the results are merged into a single json db and then that db is written in the main process using 64 xz threads.

I pushed some changes to make it clearer I think.